pkg/executor/aggfuncs/spill_deserialize_helper.go

```diff
@@ -0,0 +1,438 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aggfuncs
+
+import (
+	"github.com/pingcap/tidb/pkg/util/chunk"
+	"github.com/pingcap/tidb/pkg/util/hack"
+	util "github.com/pingcap/tidb/pkg/util/serialization"
+)
+
+type posAndBytes struct {
+	bytes []byte
+	pos   *int64
+}
+
+func newPosAndBytes() *posAndBytes {
+	pos := int64(0)
+	return &posAndBytes{pos: &pos}
+}
+
+func (p *posAndBytes) prepare(col *chunk.Column, idx int) {
```

Comment:
And gpt said that `reset` would be a better name than `prepare`

Reply:
> And gpt said that `reset` would be a better name than `prepare`

okk, I have refactored posAndBytes and `prepare` function has been deleted.